internal/services/applications/application_resource.go

```diff
@@ -225,6 +274,51 @@ func applicationResource() *schema.Resource {
 				},
 			},
 
+			"info": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"logo_url": {
+							Description: "CDN URL to the application's logo",
+							Type:        schema.TypeString,
+							Computed:    true,
+						},
+
+						"marketing_url": {
+							Description: "URL of the application's marketing page",
+							Type:        schema.TypeString,
+							Optional:    true,
+						},
+
+						"privacy_statement_url": {
+							Description: "URL of the application's privacy statement",
+							Type:        schema.TypeString,
+							Optional:    true,
+						},
+
+						"support_url": {
+							Description: "URL of the application's support page",
+							Type:        schema.TypeString,
+							Optional:    true,
+						},
+
+						"terms_of_service_url": {
+							Description: "URL of the application's terms of service statement",
+							Type:        schema.TypeString,
+							Optional:    true,
+						},
+					},
+				},
+			},
+
+			"oauth2_post_response_required": {
+				Description: "Specifies whether, as part of OAuth 2.0 token requests, Azure AD allows POST requests, as opposed to GET requests.",
```

Comment:
this says "post allowed" vs the property name "response_required"

Reply:
This is a tricky one since the docs are ambiguous on the behaviour. I'll try to dig up the actual meaning and make it clearer one way or the other.

Reply:
It seems like this description, which was pulled from [application manifest docs](https://docs.microsoft.com/en-gb/azure/active-directory/develop/reference-app-manifest?WT.mc_id=Portal-Microsoft_AAD_RegisteredApps#oauth2requiredpostresponse-attribute), is the only documentation available on this one. Will follow up separately and try to find out more.

Reply:
any update on this one?

Reply:
Not as yet, have reached out to get some clarity on this field

Reply:
I've raised this separately as #491 against the 2.0.0 milestone in order to unblock the PR stack :)